<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="apply_filter_btn">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="move_btn">
              <property name="text">
               <string>Move</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="delete_btn">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="selected_combo_box">
              <item>
               <property name="text">
                <string>Selected</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>All zeros</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>All poles</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Both</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="conjugate_check_box">
              <property name="text">
               <string>Add Conjugates</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="PlotWidget" name="z_plane_plot"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="PlotWidget" name="magnitude_plot"/>
            </item>
            <item>
             <widget class="PlotWidget" name="phase_plot"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Mouse Input</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="upload_signal_btn">
              <property name="text">
               <string>Upload Signal</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" rowspan="2">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QComboBox" name="all_pass_lists"/>
            </item>
            <item>
             <widget class="QGraphicsView" name="all_pass_graph"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget" native="true">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="PlotWidget" name="graphicsView_2"/>
          </item>
          <item>
           <widget class="PlotWidget" name="graphicsView"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="open_btn">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
